### DISCUSSION TITLE
Is Playback/Viewer zoom (not timeline zoom) a thing?

### DISCUSSION THREAD

**Discussion by cmdarrow** [mifi/lossless-cut#1765] [Q&A]
I was curious if there is a way to zoom in/out on the video playback/viewer (not the timeline). It seems to default to "fit to width" and I'm wondering if there are options or key/mouse strokes to modify this just so I can get a closer look at a certain spot in a frame while cutting footage. Am I missing something obvious or is this not a feature that exists at this time? Even if I could just force a "fill window" or "fit to height" when needed, that would be sufficient. Loving this software so far, by the way! Has proven to be an extremely helpful addition to my video processing toolkit and I'm very impressed with how light/nimble it seems to be. Thank you!

**Comment by mifi** [comment]
currently not possible. see #779

**Reply by cmdarrow** [reply]
Understood, thank you for your response and the great app! I just wanted to be sure I wasn't missing something obvious.
